### PR TITLE
docs: Update the Golang CI configurations section

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
## Description

Update the Golang CI configurations section as follows:

- Specify the type of secret required to be created, as GitHub now has two kinds of secrets under **Secrets and variables** - **Environment and Repository secret.**
- Improve the context of a couple of sentences.

## Reviews

- [x] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by dvdksn — DISMISSED** [review]
LGTM

File: content/language/golang/configure-ci-cd.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ Create a GitHub repository, configure the Docker Hub secrets, and push your sour
 2. Open the repository **Settings**, and go to **Secrets and variables** >
    **Actions**.
 
-3. Create a new secret named `DOCKER_USERNAME` and your Docker ID as value.
+3. Create a new **Repository secrets** named `DOCKER_USERNAME` and your Docker ID as value.
 
-4. Create a new [Personal Access Token (PAT)](../../security/for-developers/access-tokens.md/#create-an-access-token) for Docker Hub. You can name this token `docker-tutorial`.
+4. Create a new [Personal Access Token (PAT)](../../security/for-developers/access-tokens.md/#create-an-access-token) for Docker Hub. You can name this token `docker-tutorial`. Make sure access permissions include Read, Write and Delete.
```
Comment (crazy-max) [inline content/language/golang/configure-ci-cd.md:30]: Don't think it needs `Delete` perm

Comment (Pradumnasaraf) [reply content/language/golang/configure-ci-cd.md]: Yes, you are correct. I have updated it. Thank you for the review.

**Review by dvdksn — APPROVED** [review]

**Review by crazy-max — APPROVED** [review]
Thanks!

**Pull request merged**